README.md

```diff
@@ -573,441 +468,19 @@ debugger attaches, and you can then step through the code.
 mvn clean test -Dtest=ServerCommandTest -Dmaven.surefire.debug="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000 -Xnoagent -Djava.compiler=NONE"
 ```
 
-#### Potentially speed up the development cycle
+#### Potentially Speed Up the Development Cycle
 
 The above commands all have 'clean' in the command.  It is safest to do that.  But it is 
 possible to omit the 'clean' and maven will then only compile files that have changed.  This can
 speed up things, but for Emissary it is unlikely to speed up much.
 
-### IDE integration
```

Comment:
Moved to all Emissary Commands info into a separate README, since there was so much content here. The new README is linked to above so users can easily navigate to it